pages/index.tsx

```diff
@@ -37,7 +42,11 @@ export default function Home() {
         setHasMore(true);
       } else if (posts.length === 0) {
         setHasMore(true);
```

Comment:
Why?

Reply:
This is know that there is no more post at the end of the posts

Reply:
But how does no post (posts.length===0) equate to there being more posts?